src/App.tsx

```diff
@@ -1,24 +1,128 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect, useMemo } from 'react';
 import { UserWarning } from './UserWarning';
+import { FormTodo } from './component/FormTodo/FormTodo';
+import { MainTodo } from './component/MainTodo/MainTodo';
+import { Footer } from './component/Footer/Footer';
+import { Error } from './component/Error/Error';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import { client } from './utils/fetchClient';
+import { FilterByWords } from './types/enums';
 
-const USER_ID = 0;
+const USER_ID = 10599;
 
 export const App: React.FC = () => {
+  const [todos, setFormValue] = useState<Todo[]>([]);
+  const [filterStatus, setFilterStatus] = useState(FilterByWords.All);
+  const [error, setError] = useState<string>('');
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [isInputDisabled, setIsInputDisabled] = useState(false);
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then(setFormValue)
+      .catch((err: Error) => {
+        setError(err.message);
+      });
+  }, []);
+
+  useEffect(() => {
+    if (tempTodo !== null) {
+      setTempTodo(null);
+    }
+  }, [todos]);
```

Comment:
temp todo should be set to null after the request with add new todo finalized (`.finally` clause to a post request)